libs/blocks/media/media.js

```diff
@@ -30,9 +30,10 @@ function decorateAvatar(el) {
 function decorateQr(el) {
   const text = el.querySelector('.text');
   if (!text) return;
-  const appStore = text.children[(text.children.length - 1)];
-  const googlePlay = text.children[(text.children.length - 2)];
   const qrImage = text.children[(text.children.length - 3)];
+  const qrCodeLinks = el.querySelectorAll('a');
```

Comment:
Should we not stick to `text.querySelectorAll('a')`?

Reply:
Implemented your suggestion with the querySelector below